src/vm/docs/mod.rs

```diff
@@ -91,6 +91,20 @@ const NONE_KEYWORD: KeywordAPI = KeywordAPI {
 "
 };
 
+const TO_UINT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(to-uint i)",
+    description: "Tries to convert the `int` argument to a `uint`. Will cause a runtime error and abort if the supplied argument is negative.",
+    example: "(to-uint 238) ;; Returns u238"
+};
+
+const TO_INT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(to-int u)",
+    description: "Tries to convert the `uint` argument to an `int`. Will cause a runtime error and abort if the supplied argument is >= 2^127.",
```

Comment:
Should be `2^127 - 1`.  Also, to me, `^` means `bitwise xor`.  Do you think `**` would make this less ambiguous, or more confusing?  Maybe `pow(2, 127)`?

Reply:
So, the choices are:

* `^`
* `**`
* `pow(int, int)`

If I am correct, I think the last is more readable. The last is actually similar to the Javascript language `Math.pow` another positive. @jcnelson @kantai

Reply:
Great, that's what I thought as well